tests/test_base64.rs

```diff
@@ -32,8 +32,7 @@ fn test_garbage() {
     new_ucmd()
         .arg("-d")
         .pipe_in(input)
-        .fails()
-        .stderr_only("base64: error: invalid character (Invalid character '0' at position 20)\n");
```

Comment:
If these are GNU error messages, good job on removing them, if not please leave them in.